proxy/envoy/testdata/rds-fault.json.golden

```diff
@@ -0,0 +1,62 @@
+{
+  "virtual_hosts": [
+   {
+    "name": "hello.default.svc.cluster.local:http",
+    "domains": [
+     "hello:80",
+     "hello",
+     "hello.default:80",
+     "hello.default",
+     "hello.default.svc:80",
+     "hello.default.svc",
+     "hello.default.svc.cluster:80",
+     "hello.default.svc.cluster",
+     "hello.default.svc.cluster.local:80",
+     "hello.default.svc.cluster.local",
+     "10.1.0.0:80",
+     "10.1.0.0",
+     "10.1.1.0:80",
+     "10.1.1.0"
+    ],
+    "routes": [
+     {
+      "prefix": "/",
+      "cluster": "inbound:80"
+     }
+    ]
+   },
+   {
+    "name": "world.default.svc.cluster.local:http",
+    "domains": [
+     "world:80",
+     "world",
+     "world.default:80",
+     "world.default",
+     "world.default.svc:80",
+     "world.default.svc",
+     "world.default.svc.cluster:80",
+     "world.default.svc.cluster",
+     "world.default.svc.cluster.local:80",
+     "world.default.svc.cluster.local",
+     "10.2.0.0:80",
+     "10.2.0.0"
+    ],
+    "routes": [
+     {
+      "prefix": "/",
+      "cluster": "outbound:world.default.svc.cluster.local:http:version=v1",
```

Comment:
One suggestion. Can we sync this version=v0 with the header value or (use arbitrary header value scooby: doo), so that it doesn't get confusing? I realize that this is how the integration tests are done, but if we are adding test cases, this might be an opportunity to fix. WDYT?